packages/framework/pr_tools/trilinosprhelpers/TrilinosPRConfigurationBase.py

```diff
@@ -649,46 +649,13 @@ def create_package_enables_file(self, dryrun=False):
         return 0
 
 
-    def validate_branch_constraints(self):
```

Comment:
Is there a replacement for this safeguard?

Reply:
I'm not really that concerned about this one.  The repository permissions in GitHub should forbid anybody other than the autotester, you, and Jim W. at the moment.

Reply:
And realistically I think I'm going to change them to be "only the autotester"

Reply:
Okay, so I changed that.  Now that I'm thinking about it though, there may still be an issue with the autotester being able to merge if somebody uses AT: AUTOMERGE.  I'll see what I can come up with.  Really I want to set it up so that only the autotester can even _open_ PRs into master.

Reply:
> Although, on second thought, what is the issue with direct merges to master? As long as it passes the correct status checks, I don't see why it should be an issue.

The makes sense. The main point is to assure that those check pass before a merge is allowed. Allows for a lot of upstream messiness.

Reply:
Yes, the autotester system seems to always test changes into master with the correct set of builds (the way I'm reading the config files), so I think it's safe to remove this from a code quality standpoint.  Worst case, somebody decides to merge to the master branch, and the history isn't as clean as we want it.  However, I think it's reasonable to rely on code review to catch that particular issue.  Does that satisfy the concern of removing this check?

Reply:
I would say so and if it becomes a problem address it as needed.